src/pages/sighting/SightingCore.jsx

```diff
@@ -79,6 +81,17 @@ export default function SightingCore({
     },
     [data, fieldSchemas],
   );
+  const [
+    messageForConfirmDelete,
+    setMessageForConfirmDelete,
+  ] = useState(null);
+
+  useEffect(() => {
+    const message = vulnerableIndividual
+      ? 'SIGHTING_DELETE_VULNERABLE_INDIVIDUAL_MESSAGE'
+      : 'CONFIRM_DELETE_SIGHTING_DESCRIPTION';
+    setMessageForConfirmDelete(message);
+  }, vulnerableIndividual);
```

Comment:
The message does not need to be stored in state. It can just be calculated when the function runs.

Reply:
I was still having trouble wrapping my head around react lifecycle stuff sometimes. It clearly works without storing in state, exactly as you've indicated here. Thank you for prompting me to re-read a little bit about when components re-render lol. I feel more comfortable with why this was unnecessary now.